processor/src/tasks/writeKitspaceYaml.ts

```diff
@@ -1,59 +1,57 @@
-import escape from 'escape-html'
-import LinkifyIt from 'linkify-it'
 import path from 'node:path'
 
+import { unified } from 'unified'
+import rehypeHighlight from 'rehype-highlight'
+import rehypeSanitize from 'rehype-sanitize'
+import rehypeStringify from 'rehype-stringify'
+import remarkEmoji from 'remark-emoji'
+import remarkGfm from 'remark-gfm'
+import remarkParse from 'remark-parse'
+import remarkRehype from 'remark-rehype'
+
 import { JobData } from '../jobData.js'
 import * as utils from '../utils.js'
 
-const linkify = new LinkifyIt()
-
-export default function writeKitspaceYaml(
+export default async function writeKitspaceYaml(
   job,
   { kitspaceYaml, outputDir }: Partial<JobData>,
 ) {
   const kitspaceYamlJson = path.join(outputDir, 'kitspace-yaml.json')
   job.updateProgress({ status: 'in_progress', file: kitspaceYamlJson })
-  const KitspaceYamlJsonLinkified = linkifyKitspaceYaml(kitspaceYaml)
+
+  const rendered = await renderKitspaceYamlSummaries(kitspaceYaml)
 
   return utils
-    .writeFile(kitspaceYamlJson, JSON.stringify(KitspaceYamlJsonLinkified, null, 2))
+    .writeFile(kitspaceYamlJson, JSON.stringify(rendered, null, 2))
     .then(() => job.updateProgress({ status: 'done', file: kitspaceYamlJson }))
     .catch(error =>
       job.updateProgress({ status: 'failed', file: kitspaceYamlJson, error }),
     )
 }
 
-function linkifyKitspaceYaml(kitspaceYaml) {
+async function renderKitspaceYamlSummaries(kitspaceYaml) {
   if (kitspaceYaml.multi) {
-    const linkifiedKitspaceYaml = kitspaceYaml
-    Object.keys(kitspaceYaml.multi).forEach(subProject => {
-      linkifiedKitspaceYaml.multi[subProject] = linkifyProjectSummary(kitspaceYaml.multi[subProject])
-    })
-    return linkifiedKitspaceYaml
-  }
-
-  return linkifyProjectSummary(kitspaceYaml)
-}
-
-function linkifyProjectSummary(kitspaceYaml) {
-  let escapedSummary = escape(kitspaceYaml.summary || '')
-  const matches = linkify.match(escapedSummary)
-
-  if (matches) {
-    for (const match of matches) {
-      // Use https by default
-      const url = new URL(match.url)
-      if (!match.schema) {
-        url.protocol = 'https:'
+    const rendered = { multi: {} }
+    for (const key of kitspaceYaml.multi) {
```

Comment:
```suggestion
    for (const key of Object.keys(kitspaceYaml.multi)) {
```
![e6f236539a1d1ddf02cb6d1884407454-3427410689](https://user-images.githubusercontent.com/206854/198689852-7058fb41-bea8-4905-bcad-45bcc84fd44a.gif)